tests/__main__.py

```diff
@@ -1,3 +1,7 @@
+# Needs investigation, pylint was running tests
+# Could be `pylint_pytest` plugin, skipping check for this file for now
+# pylint: skip-file
+
```

Comment:
I am not sure why, but when not skipping this file, `pylint` was running the tests for some reason.